playground/frontend/lib/l10n/app_en.arb

```diff
@@ -154,5 +154,41 @@
   "clearOutput": "Clear Output",
   "@clearOutput": {
     "description": "Title for the Clear Output shortcut row"
+  },
+  "pipelineOptions": "Pipeline Options",
+  "@pipelineOptions": {
+    "description": "Title for the Pipeline Options"
+  },
+  "rawPipelineOptions": "Raw",
+  "@rawPipelineOptions": {
+    "description": "Title for the Raw Pipeline Options Tab"
+  },
+  "optionsPipelineOptions": "Options",
+  "@optionsPipelineOptions": {
+    "description": "Title for the Options Pipeline Options Tab"
+  },
+  "saveAndClose": "Save & close",
+  "@saveAndClose": {
+    "description": "Text for save and close button on pipeline dropdown"
+  },
+  "addPipelineOptionParameter": "Add parameter",
+  "@addPipelineOptionParameter": {
+    "description": "Text for add parameter button on pipeline dropdown"
+  },
+  "input": "Input",
+  "@input": {
+    "description": "Text input label"
+  },
+  "name": "Name",
+  "@name": {
+    "description": "Text name label"
+  },
+  "value": "Value",
+  "@value": {
+    "description": "Text value label"
+  },
+  "pipelineOptionsError": "Please check the format (example: --key1 value1 --key2 value2), only alphanumeric and \",*,/,-,:,;,',. symbols are allowed",
```

Comment:
Nit: Perhaps this would be better as `(example: [key1, value1], [key2, value2])` to match the new UI. The dashes might be a bit confusing as they no longer exist.

Reply:
I think dashes are more suitable here because they are like command-line arguments and users are familiar with them.